test/KHI_growthRate/include/picongpu/param/grid.param

```diff
@@ -45,7 +45,6 @@ namespace picongpu
 {
     namespace SI
     {
-
```

Comment:
Not related to this PR - but fine with me

Reply:
Ohh, this should not be changed.